WooCommerce/Classes/ViewRelated/Orders/Order Details/Address Edit/EditOrderAddressForm.swift

```diff
@@ -216,16 +214,37 @@ struct EditOrderAddressForm<ViewModel: AddressFormViewModelProtocol>: View {
                 .padding(.horizontal, insets: geometry.safeAreaInsets)
                 .background(Color(.systemBackground))
 
-                Group {
-                    TitleAndToggleRow(title: viewModel.toggleTitle, isOn: $viewModel.fields.useAsToggle)
+                if viewModel.showAlternativeUsageToggle {
+                    TitleAndToggleRow(title: viewModel.alternativeUsageToggleTitle, isOn: $viewModel.fields.useAsToggle)
                         .padding(.horizontal, Constants.horizontalPadding)
                         .padding(.vertical, Constants.verticalPadding)
+                        .padding(.horizontal, insets: geometry.safeAreaInsets)
+                        .background(Color(.systemBackground))
                 }
-                .padding(.horizontal, insets: geometry.safeAreaInsets)
-                .background(Color(.systemBackground))
             }
             .background(Color(.listBackground))
             .ignoresSafeArea(.container, edges: [.horizontal, .bottom])
+
+            // Go to edit country
+            LazyNavigationLink(destination: FilterListSelector(viewModel: viewModel.createCountryViewModel()), isActive: $showCountrySelector) {
+                EmptyView()
+            }
+
+            // Go to edit state
+            LazyNavigationLink(destination: FilterListSelector(viewModel: viewModel.createStateViewModel()), isActive: $showStateSelector) {
+                EmptyView()
+            }
+
+            ///
+            /// iOS 14.5 has a bug where
+            /// Pushing a view while having "exactly two" navigation links makes the pushed view to be popped when the initial view changes its state.
+            /// EG: AddressForm -> CountrySelector -> Country is selected -> AddressForm updates country -> CountrySelector is popped automatically.
+            /// Adding an extra and useless navigation link fixes the problem but throws a warning in the console.
+            /// Ref: https://forums.swift.org/t/14-5-beta3-navigationlink-unexpected-pop/45279
+            ///
+            NavigationLink(destination: EmptyView()) {
+                EmptyView()
+            }
```

Comment:
Does Order Address Editing keep working?

Reply:
Yes! Both simulator and device.